### DISCUSSION TITLE
Trying to create a transaction with the api returns the login page

### DISCUSSION THREAD

**Discussion by Joserichi** [firefly-iii/firefly-iii#7673] [Firefly III] [answered]
I'm trying to use the api to create a transaction. I have setup the token and so, but when I try to send the request to create the transaction, I get the login page as a response. No error, no messages, anything, just the login page, so I don't know what am I doing wrong.
This is the curl request (that postman says) i'm using:
`curl --location --request PUT 'https://<url>/api/v1/transactions' \
--header 'Authorization: Bearer <token>' \
--header 'Content-Type: application/json' \
--data '{"error_if_duplicate_hash":"true","apply_rules":"true","fire_webhooks":"true","transactions":[{"type":"transfer","amount":"1","description":"Test","source_name":"<asset1>","destination_name":"<asset2>","date":"2023-06-170T12:51:00+0200"}]}'`
Trying to get a transaction works fine, so I guess my firefly instalation is reacheable and that the token is ok. What can be the problem?

**Comment by JC5** [comment, marked as answer]
Make sure you submit the "Accept: application/json" header or Firefly III won't know what to return.
